tools/ci_build/build.py

```diff
@@ -579,6 +579,39 @@ def run_onnx_tests(build_dir, configs, onnx_test_data_dir, provider, enable_para
           run_subprocess([exe,'-x'] + cmd, cwd=cwd)
 
 
+# mkldnn temporary function for running onnx tests and model tests separately.
+def mkldnn_run_onnx_tests(build_dir, configs, onnx_test_data_dir):
```

Comment:
Is this is because of rebasing with master? Doesn't look like related to this PR.

Reply:
Exactly, I have cancelled the change.